internal/jobs/common.go

```diff
@@ -15,11 +17,16 @@ import (
 const TraceName = telemetry.TracePrefix + "internal/jobs"
 
 var ErrTypeAssertion = errors.New("type assert error")
+var ErrPanicInJob = errors.New("panic during job")
```

Comment:
No idea what linter is complaining about :thinking: 
My guess would be that it wants:
```suggestion
var (
  ErrTypeAssertion = errors.New("type assert error")
  ErrPanicInJob = errors.New("panic during job")
)
```

Reply:
Yup, now it's happy! :tada: